internal/platform/logger/logger.go

```diff
@@ -0,0 +1,95 @@
+package logger
+
+//noinspection SpellCheckingInspection
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	timestampFormat = "2006-01-02T15:04:05.999Z"
+	fieldBuildHash  = "buildHash"
+
+	// FieldCtx is a key of logged context
+	FieldCtx = "ctx"
+)
+
+// reservedFields defines all fields keys which cannot be used when
+// we logging additional fields with YaaS formatter
```

Comment:
YaaS, really :) ?

Reply:
it's in all our components :D but I will fix it.